scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,190 @@
+@import 'settings';
+
+@mixin divider-position($divider-order, $large-screen: false) {
+  @if $large-screen {
+    @if $divider-order == 1 {
+      &.p-equal-height-row--has-1st-divider::before {
```

Comment:
This was supposed to be 2 separate class names

`p-equal-height-row` - a main component class name, and `has-1st-divider` as a modifier class name.

In HTML we want this to be:

`<div class="p-equal-height-row has-1st-divider has-3rd-divider" />`

rather than

`<div class="p-equal-height-row p-equal-height-row--has-1st-divider p-equal-height-row--has-3rd-divider" />`.

While the latter is how official BEM naming would be, we don't use it in Vanilla.

Reply:
I see, I went with full BEM compliant here, this is way better and much shorter :+1: